### DISCUSSION TITLE
can we do multiple cycles as we refine the research question

### DISCUSSION THREAD

**Discussion by Ishmael56** [asreview/asreview#1546] [Q&A]
Hi! just a quick question,

before i begin i want to say thank you! this tool is amazing !! 

 i'm working on a scope review whereby i've collected citations from several databases, compiled them, removed duplicates and now after assessing most of them I found 700, some what relevant, references which i now want to submit to a second team. The question is, can the second team take the 700 relevant citations and run them again through as-review as they have a more clear inclusion/exclusion criteria? 

I've already tired to test this by exporting the relevant citations and setting up a new project with those papers, but every-time i do that, it shows all the citations as labelled, specifically "relevant". how do i make the AI forget!? 

*UPDATE* i realise it's because when i export the relevant citations theres a note on each citation which says "AS-REVIEW_relevant". So i suppose my question is can i export it without the note on each citation? 

<img width="956" alt="Screenshot 2023-10-19 at 20 15 41" src="https://github.com/asreview/asreview/assets/148491426/7763bbe6-dea7-4e80-b25a-36cd8299f1e7">

**Comment by Rensvandeschoot** [comment]
Of course! Here's your initial response with grammar corrections:

This is an excellent idea! I have seen people re-importing a dataset to continue screening with a different model or to re-label the initially excluded papers to double-check their relevance, but this is yet another great idea!

If you export the file with labels and re-import this file, ASReview will recognize the labels and use these to train the first iteration of the model. If you want to re-label these records, you need to remove the labels or rename the tag we use. You could export only the relevant records (since [this pull request](https://github.com/asreview/asreview/pull/1335) was merged) as a .xlsx or .csv file (instead of .ris) and simply rename the column used for the labels. In this [table](https://asreview.readthedocs.io/en/latest/data_format.html), you can see which labels we use, so you need to choose a different column name. If you then import this file into ASReview, it will treat all records as unseen.

Make sure to also add a few irrelevant records to help with the training stage.

If you want to convert your file back to .ris, you can use convert in [Datatools](https://github.com/asreview/asreview-datatools#data-convert).
